### DISCUSSION TITLE
鼠标点到输入框后几秒钟闪退

### DISCUSSION THREAD

**Discussion by litianyuhehe** [ywmoyue/biliuwp-lite#904] [General]
### Bug 描述

鼠标点到输入框后几秒钟闪退，包括搜索框或设置里调音量的输入框

### 操作步骤

```plain text
打开应用，鼠标点进输入框，等待几秒，程序闪退
```

### 期望结果

可以正常输入，不闪退

### 日志

[20241026.log](https://github.com/user-attachments/files/17531137/20241026.log)

### 应用来源

Github/其他Github镜像站

### 安装方式

appx直接安装

### 系统版本

其它 (需注明)

### 其它系统版本

windows 11 24H2 (Build 26120)

### 设备类型

笔记本

### 设备架构

x64

### 应用架构

x64

### 代码贡献

我更擅长发现问题

**Comment by ywmoyue** [comment]
我这边没复现，看日志中也没有记录到关键信息，怀疑可能是24H2的问题，我之后在虚拟机上测一下

**Comment by litianyuhehe** [comment]
https://github.com/user-attachments/assets/978fa966-32d2-44ee-8e7f-21ab8f140e9a

录了一段视频，就是这样。

**Comment by ywmoyue** [comment]
用24H2的虚拟机测试仍然没复现 @litianyuhehe 麻烦试试用`WinUI 2 Gallery`中的输入框测一下看是不是一样会闪退

以及可以看看系统日志中有没有相关错误信息（右键开始图标->Windows日志->应用程序->找到应用程序名称为BiliLite.exe的错误日志，如下图所示）
![image](https://github.com/user-attachments/assets/b80a6e2e-04e3-42fe-bc19-9f3b48469aa8)


https://www.microsoft.com/store/productId/9MSVH128X2ZT?ocid=pdpshare

**Comment by litianyuhehe** [comment]
@ywmoyue 是搜狗输入法的原因，微软的就不会
日志报错信息：
出错应用程序名称： isgpet.exe，版本： 1.0.2.2521，时间戳： 0x66da7f0b
出错模块名称： isgpet.exe， 版本： 1.0.2.2521，时间戳： 0x66da7f0b
异常代码： 0x80000003
错误偏移： 0x0004f7cb
出错进程 ID： 0x3B40
出错应用程序开始时间： 0x1DB27BAD1A66A20
Faulting 应用程序路径： C:\Program Files (x86)\SogouInput\Components\isgpet\1.0.2.2521\isgpet.exe
Faulting 模块路径： C:\Program Files (x86)\SogouInput\Components\isgpet\1.0.2.2521\isgpet.exe
Report ID： a7fe19ba-eae6-45ec-a15c-9bf661cd3cd7
Faulting 包全名： 5422.502643927C6AD_4.7.6.0_x64__vn31kc91nth4r
Faulting 程序包相对应用程序 ID： App 个

**Comment by ywmoyue** [comment]
这样的话，我这边是处理不了的
